llm_unlearn_ucl/unlearn_harm.py

```diff
@@ -229,25 +237,47 @@ def main(args) -> None:
     assert (
         args.samples_count // args.sequential
     ) % args.batch_size == 0, "samples in each 'sequence' (--samples_count / --sequential) should be a multiple of batch_size."
-    accelerator = Accelerator(
-        mixed_precision="bf16"
-    )  # accelerator precision can be specified if required.
+
+    if args.wandb_log:
+        accelerator = Accelerator(log_with="wandb", mixed_precision="bf16")
```

Comment:
For visibility: @Willmish's code removes bf16 from the python code and defines it in the deepspeed config.